benchmarks/python/test_huggingface_attn_bwd.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 from nvfuser import FusionDefinition, DataType
 from nvfuser.pytorch_utils import torch_dtype_to_nvfuser_dtype
-from .core import run_benchmark, clear_cuda_cache
+from .core import run_benchmark, clear_cuda_cache, unary_bwd_torch
```

Comment:
where does this `unary_bwd_torch` defined? It doesn't exist in `core.py`.

Reply:
It is defined here: https://github.com/NVIDIA/Fuser/blob/e10c4714aede47a41a1d71a879681d3f94c00df0/benchmarks/python/core.py#L142